web/cypress/e2e/countrypanel.spec.ts

```diff
@@ -21,7 +21,7 @@ describe('Country Panel', () => {
 
     cy.contains('Carbon emissions').should('have.attr', 'aria-checked', 'false');
     cy.contains('Carbon emissions').click().should('have.attr', 'aria-checked', 'true');
-    cy.contains('0 t/min');
+    cy.contains('0t/min');
```

Comment:
Random fact, a tonne is also known as a Megagram (Mg)

Reply:
> I think t for tonne is good here, Megagram is probably only known by a niche group.

I'm just worried it might get confused with the long ton (1016 kg) or the short ton (907 kg)